Sources/Math.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
```

Comment:
http://stackoverflow.com/questions/15231466/whats-the-difference-between-pi-and-m-pi-in-objc

<img width="753" alt="screen shot 2016-10-27 at 23 04 03" src="https://cloud.githubusercontent.com/assets/1088217/19785455/b468969c-9c99-11e6-84a1-dddeeda6b73e.png">